aws_lambda_powertools/event_handler/middlewares/openapi_validation.py

```diff
@@ -1,13 +1,13 @@
+from __future__ import annotations
+
 import dataclasses
 import json
 import logging
 from copy import deepcopy
-from typing import Any, Callable, Dict, List, Mapping, MutableMapping, Optional, Sequence, Tuple
+from typing import TYPE_CHECKING, Any, Callable, Mapping, MutableMapping, Sequence
 
 from pydantic import BaseModel
 
-from aws_lambda_powertools.event_handler import Response
-from aws_lambda_powertools.event_handler.api_gateway import Route
 from aws_lambda_powertools.event_handler.middlewares import BaseMiddlewareHandler, NextMiddleware
 from aws_lambda_powertools.event_handler.openapi.compat import (
     ModelField,
```

Comment:
Should `ModelField` be imported in TYPE_CHECKING section? Same with `NextMiddleware`?

Reply:
Oh, but looks like you're right, these are not being used in a pydantic model in this code. I was relying mostly on ruff to detect what needs to go to TYPE_CHECKING, and it seems to have missed these.